src/amazon/AmazonProduct.js

```diff
@@ -12,6 +14,8 @@ class AmazonProduct {
     this.ASIN = product.Identifiers.MarketplaceASIN.ASIN;
     this.bestSalesRanking = this._getBestSalesRanking(product.SalesRankings);
     this.upc = UPC ? UPC : 'UNKNOWN'
+    this.category = product.AttributeSets['ns2:ItemAttributes']['ns2:ProductGroup'];
+    this.MIN_SELL_PRICE = 10; // $10
```

Comment:
Instead of using the minimum sell price, could we use a certain %ROI instead? Here, using SELL price from Amazon can be limiting. If AMZN sells for $9 and walmart sells for $2, there is potential to make about 4-5 dollars. If this is a highly ranked (popular item), it can be a regular stream for profit. Let's use an ROI here instead. I implemented that in the simple_analysis fix

Reply:
This is looking the price an item sells on Amazon. $3 is about the lowest you will pay for shipping so if you spend $3 shipping from walmart to your house and then another $3 shipping to whoever buys it on amazon, that means in order to make any sort of profit, you would have had to buy the item for $4 from walmart. Also considering that amazon keeps part of the profit, you would really have to have bought it for $3 on walmart.
In other words, the best profit you can hope to make from selling an object for $10 on Amazon is $3. Most cases will be less, so I think this heuristic is fine as it is removing a lot of products that can't possibly yield profit.

Reply:
Good explanation. Here are a few things to note:
1. We currently have a way to calculate shipping cost from Walmart and there are cases when shipping is free ($0.00). 
2. Selling as an FBA (Fulfillment by Amazon) on Walmart makes it that we don't ship to the seller. We ship to Amazon at a discounted price (because we are paying $40 monthly) and Amazon ships to buyer. We might incur cost for items spending too long on shelf but we can reduce that by buying items that we sell.
So, it is possible that an item is selling $10 on Amazon and it costs us less than $5 to get it to the client, so we can still make profit. I also think making $3 on a highly ranked item that sells in huge quantities isn't a bad idea. It is free money and once we figure out how to send items to prep center, we won't even have to do anything but buy and make profit.

In summary, I agree with eliminating low profit items but I don't think a cut-off price is ideal. We could use %ROI after factoring all costs. I will take 50% profit for each $10 I spend instead of nothing.

Reply:
Followed up with your suggestion. https://github.com/sulaimonlasisi/clearance-app/pull/19/commits/aebcbda6a12bc532b6471c3858b26b99861cfe00